.github/workflows/check.yml

```diff
@@ -56,10 +56,18 @@ jobs:
           fi
 
       - name: "Install: Rust stable toolchain"
-        uses: dtolnay/rust-toolchain@stable
+        run: rustup toolchain install stable --component llvm-tools-preview --target wasm32-unknown-unknown
```

Comment:
https://github.com/dtolnay/rust-toolchain#inputs

Reply:
Hmm, I'm not sure if I should use actions for this because they override the toolchain